sdk/storage/Directory.Build.props

```diff
@@ -0,0 +1,87 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+  Import our Directory.Build.props and /eng infrastructure
+  -->
+  <PropertyGroup>
+    <!--
+    We're currently on 1.0.0-preview.X and we'll share this version number
+    across all Storage projects until the need to version independently arises.
+    See /eng/Versioning.props for more details.
+    -->
+    <VersionPrefix>1.0.0</VersionPrefix>
+
+    <!--
+    The Client SDK Analyzers are based on Design Guidelines that haven't been
+    fully formalized so we won't enable them just yet.
+     -->
+    <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
+
+    <!--
+    The Azure Storage projects don't want the Rest Client Runtimes or default
+    test projects (xunit instead of MSTest).
+    -->
+    <ImportDefaultReferences>false</ImportDefaultReferences>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />  
+
+
+  <PropertyGroup>
+    <!-- We'll target the standard SDK frameworks -->
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+
+    <!-- Note that we disable this for DEBUG -->
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <!-- Have the compiler generate fully qualified paths for error messages -->
+    <GenerateFullPaths>true</GenerateFullPaths>
+    
+    <DebugSymbols>true</DebugSymbols>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+
+    <!--
+    Storage used to point to its main documentation page, but the rest of the
+    Azure .NET SDK projects point to the github repo instead.  I'm leaving this
+    commented out for now so we can change it back if required.
+
+    <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkId=235168</PackageProjectUrl>
+    -->
+  </PropertyGroup>
+
+
+  <!-- DEBUG specific settings -->
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <!-- Treat warnings as errors, except in Debug -->
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
We should probably treat warnings as errors for all configurations.

Reply:
Yes, I agree.  I made the change, but we can flip it back if folks don't like it.

Reply:
I'm good with it.